website/content/docs/commands/operator/debug.mdx

```diff
@@ -55,6 +55,9 @@ true.
 
 - `-log-level=DEBUG`: The log level to monitor. Defaults to `DEBUG`.
 
+- `-log-include-location`: Include file and line information in each log line
+  monitored. The default is `true`.
```

Comment:
Does `-log-include-location=false` work to disable this flag? And would we ever want to? I can't think of any downside of a hardcoded `true` value here, but also can't see why an extra flag would hurt 🤷

Reply:
Yeah I tested this locally and it does disable the log location as expected. I don't expect it would ever be used, but adding it was trivial 🤷🏻